pkg/flow/internal/controller/node_config_import.go

```diff
@@ -0,0 +1,40 @@
+package controller
+
+import (
+	"github.com/grafana/river/ast"
+	"github.com/grafana/river/vm"
+)
+
+// TODO: This struct is an empty shell for now, its implementation will come via another PR.
+
+type ImportConfigNode struct {
```

Comment:
I would ask that we isolate things out more. The ask was for the larger PR to be split into smaller, isolated PRs, so I wouldn't expect to see Import referenced at all here. 

I also don't understand what "ImportConfigNodesChildren" means yet, so I would ask that all code related to import be removed from this PR.